components/hackathon/common/HackathonWinners.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Grid, Icon, Image, Link, VStack } from "@chakra-ui/react";
+import { FiExternalLink } from "react-icons/fi";
+import { Heading, Text } from "tw-components";
+
+interface IWinner {
+  name: string;
+  image: string;
+  link: string;
+  position: string;
+}
+
+interface HackathonWinnersProps {
+  winners: IWinner[];
+}
+
+export const HackathonWinners: React.FC<HackathonWinnersProps> = ({
+  winners,
+}) => {
+  return (
+    <VStack
+      mb={{
+        base: 40,
+        md: 0,
+      }}
+    >
+      <Heading size="title.2xl" textAlign="center" paddingTop={10}>
+        Hackathon Winners
+      </Heading>
+      <Grid
```

Comment:
better if this is a SimpleGrid, otherwise we get this different sized cards
![image](https://user-images.githubusercontent.com/50103937/212752061-d5a216cd-2ee9-46e8-9a38-24dd8c896e67.png)